src/components/TransactionBlock/AssetBridgeTransactionBlock.tsx

```diff
@@ -273,6 +274,7 @@ const AssetBridgeTransactionBlock = ({
     <RouteOption
       route={availableRoutes?.find((route) => route.id === option.value)}
       isChecked={selectedRoute?.value && selectedRoute?.value === option.value}
+      cost={option.gasCostUSD && `$ ${option.gasCostUSD}`}
```

Comment:
additionally consider using formatAmountDisplay with minimumFractionDigits: 2 as per latest product requirements we were asked to show 2 decimals on USD price
```suggestion
      cost={option.gasCostUSD && `$${option.gasCostUSD}`}
```